reports/report_details.py

```diff
@@ -681,14 +681,8 @@ def add_transnational_to_regions(regions):
             'name': '42',
             'title': 'Stories where issues of gender equality/inequality are raised by region',
             'desc': '',
-            'reports': [],
-            'historical': '42',
-        },
-        '2010': {
-            'name': '42',
-            'title': 'Breakdown of Stories where issues of gender equality/inequality are raised by region',
-            'desc': 'Stories where issues of gender equality/inequality are raised by region',
             'reports': ['global'],
```

Comment:
What's going on here? `42` should be there in **2010**.

Reply:
I might have missed that.

Reply:
actually, it's not. Sheet 42 looks like it was introduced in 2015.